src/digitKeyboard.js

```diff
@@ -0,0 +1,33 @@
+"use strict";
+
+import { EventDispatcher } from './eventDispatcher.js';
+
+/**
+ *  DigitKeyboard sets the digits on the screen.
+ */
+export class DigitKeyboard extends EventDispatcher {
+
+    /**
+     *  Constructor.  
+     */
+    constructor() {
+        super();
+        this._bindEvents();
+    }
+
+    /**
+     * Binds event listeners to digit buttons.
+     */
+    _bindEvents() {
+        let buttons = document.querySelector('.allButtons');
+        let numberButtons = buttons.querySelectorAll('.number');
+
+        for (let i = 0; i < numberButtons.length; i++) {
+            numberButtons[i].addEventListener("click", function(event) {
```

Comment:
Now we have a proper solution, but ES5! Could you update your solution so that it is ES6 style?